.github/workflows/health_checks.yml

```diff
@@ -307,6 +307,7 @@ jobs:
       - name: Validate that change set has necessary dependency updates
         run: |
           npx changeset version
+          npm update
```

Comment:
This check was failing with https://github.com/aws-amplify/amplify-backend/actions/runs/8840417515/job/24275784441#step:6:48
which appears to be because after running changeset version the versions in the package-lock were incompatible with the new versions. Running npm update updates the package-lock with the bumped versions before performing the dependency check